devices/android/device-base

```diff
@@ -0,0 +1,47 @@
+{% set PROJECT = PROJECT|default("testcases/android/") %}
+{% extends PROJECT + "master.jinja2" %}
+
+{% set partitions_urls = partitions_urls|default({}) %}
+{% set partitions_names = partitions_names|default({}) %}
+{% set timeout_download = timeout_download |default(10) %}
+
+{% block download_target %}
+{% if partitions_urls %}
+- deploy:
+    timeout:
+      minutes: {{timeout_download}}
+    to: downloads
+    images:
+      {% block partitions_urls %}
+      {% for partition in partitions_urls %}
+      {{partition}}:
+        url: {{partitions_urls[partition]}}
+        {% if partitions_urls[partition].endswith("xz") %}
+        compression: xz
+        {% endif %}
+      {% endfor %}
+      {% endblock partitions_urls %}
+{% endif %}
+{% endblock download_target %}
```

Comment:
how much of this can be pulled in from the fastboot file?

Reply:
you mean the root fastboot.jinja2 file? if it's one include file, then we could think about using it...
maybe we could do some changes in the future on the deploy_target to make it more generic that can be used everywhere, but it's not something at this time.

Reply:
ok